### ISSUE TITLE
Ability to configure ephemeral resource limits on pods

### ISSUE THREAD

**Issue by FleetAdmiralButter** [uselagoon/remote-controller#204]
Hi,

We've recently run into an issue where a project deployed on Lagoon was able to cause a worker node to stop working properly by filling up the `/tmp` folder with large files. 

This caused the root filesystem on the node to fill up and it went into DiskPressure, causing pods to get evicted and rescheduled.

Would you consider adding a feature to Lagoon where deployed pods have `spec.containers[].resources.limits.ephemeral-storage` configured with a default value?

Thanks!

**Comment by shreddedbacon** [comment]
Ah, this sounds like a pretty decent thing to configure. But you're talkin about the running environment pods, not build pods yeah?

**Comment by FleetAdmiralButter** [comment]
@shreddedbacon Yeah - this issue was caused by a running CLI pod and not a build pod.

**Comment by shreddedbacon** [comment]
Ok, I'm going to transfer this to the `build-deploy-tool` repo where the work to implement this will be done. The result will be similar to this PR: https://github.com/uselagoon/build-deploy-tool/pull/120
